integration_tests/pages/page.ts

```diff
@@ -4,23 +4,19 @@ import logAccessibilityViolations from '../support/logAccessibilityViolations'
 export type PageElement = Cypress.Chainable<JQuery>
 
 export default abstract class Page {
-  static verifyOnPage<T>(constructor: new () => T): T {
-    return new constructor()
+  static verifyOnPage<T, Options>(constructor: new (options: Options) => T, options?: Options): T {
```

Comment:
Using generics here so that any (optional!) options are typed correctly so it can be flexible for requirements of different individual page object constructors.